js/views/TooltipView.js

```diff
@@ -0,0 +1,96 @@
+import { templates } from 'core/js/reactHelpers';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+export default class TooltipView extends Backbone.View {
+
+  attributes() {
+    return {
+      'aria-live': 'assertive'
+    };
+  }
+
+  className() {
+    return 'tooltip__container';
+  }
+
+  /**
+   * @param {TooltipModel} tooltip
+   * @param {jQuery} $mouseoverEl
+   */
+  show(tooltip, $mouseoverEl) {
+    this.model = tooltip;
+    this.$target = $mouseoverEl;
+    this.$el.addClass('is-loading is-shown');
+    this.listenTo(this.model, 'all', this.render);
+    this.render();
+    this.position($mouseoverEl);
+    this.$el.removeClass('is-loading');
+  }
+
+  hide() {
+    this.stopListening(this.model);
+    this.model = null;
+    this.$target = null;
+    this.$el.removeClass('is-shown');
+  }
+
+  render() {
+    const Template = templates.tooltip;
+    ReactDOM.render(<Template {...this.model.toJSON()} />, this.el);
+  }
+
+  position($mouseoverEl) {
+    const targetBoundingRect = this.$target[0].getBoundingClientRect();
+
+    const availableWidth = $('html')[0].clientWidth;
+    const availableHeight = $('html')[0].clientHeight;
```

Comment:
can add the optional chaining operator to the available width and available height constants too

```suggestion
    const availableWidth = $('html')[0]?.clientWidth;
    const availableHeight = $('html')[0]?.clientHeight;
```